docs/v3/codecs/sharding-indexed/v1.0.rst

```diff
@@ -155,6 +156,11 @@ Sharding can be configured per array in the :ref:`array-metadata` as follows::
     be used for index codecs. It is RECOMMENDED to use a little-endian codec 
     followed by a crc32c checksum as index codecs.
 
+``index_location``
```

Comment:
For backwards compatibility with existing data we should say that if not present, defaults to "end".